### PR TITLE
fixes broken user-pref path

### PR DESCRIPTION
**Pull request by InsaneZein**
depends on https://github.com/RedHatInsights/user-preferences-frontend/pull/179

### CODE REVIEW

**Review by aferd — APPROVED** [review]

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]
Can you elaborate a bit on how exactly are the links broken? The changes in the linked PR do not change any routing. The `appUrl` affects only local development but has no effect on routing in deployed environment.

Also, I don't see any link changes in the navigation files that still reference the `/user-preferences/*` pathnames.

**Comment by InsaneZein** [comment]
@Hyperkid123 right now in stage the user preferences link directs to a blank page, because of my previous attempt to change the user preferences url. This fixes that

File: static/beta/prod/modules/fed-modules.json [hunk 1]
```diff
@@ -17,20 +17,20 @@
...
+        "defaultDocumentTitle": "Notification Preferences | User Preferences",
         "modules": [
             {
-                "id": "user-preferences-notifications",
+                "id": "my-notification-preferences-notifications",
                 "module": "./RootApp",
                 "routes": [
                     {
-                        "pathname": "/user-preferences"
+                        "pathname": "/my-notification-preferences"
```
Comment (karelhala) [inline static/beta/prod/modules/fed-modules.json:27]: We should not change the URL as it requires additional changes in Akamai redirects. The SPUR is only about updating the titles.

Comment (InsaneZein) [reply static/beta/prod/modules/fed-modules.json:27]: The ticket for the SPUR has a comment talking about updating the URL https://issues.redhat.com/browse/RHCLOUD-29464

Comment (InsaneZein) [reply static/beta/prod/modules/fed-modules.json]: I've gone ahead and reverted the URL changes.

**Comment by Hyperkid123** [comment]
@karelhala I'll leave the decision on you :)

**Review by karelhala — APPROVED** [review]

**Pull request merged**